plugins/notion/src/components/CheckboxTexfield.tsx

```diff
@@ -12,22 +12,33 @@ export function CheckboxTextfield({ value, disabled, checked: initialChecked, on
     const [checked, setChecked] = useState(initialChecked)
 
     const toggle = () => {
-      if (disabled) return
+        if (disabled) return
 
-      setChecked(!checked)
-      onChange()
+        setChecked(!checked)
+        onChange()
     }
 
     return (
-        <div className={classNames("flex bg-tertiary rounded-[8px] items-center pl-[10px]", disabled && "opacity-50 cursor-default")} onClick={toggle} role="button">
+        <label
```

Comment:
This feature was broken. The onClick on the div causes the checkbox to be toggled twice immediately. Turning the div in a label ensures that the checkbox is toggled when the whole thing is clicked.